.mocharc.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
     './app/scripts/controllers/app-state.test.js',
     './app/scripts/controllers/permissions/**/*.test.js',
     './app/scripts/controllers/mmi-controller.test.ts',
+    './app/scripts/controllers/metametrics.test.js',
```

Comment:
question: if we have migrated this test, why are we adding to the ignore section?

Reply:
The are the file `ignore` files, i.e. the files that should be ignore when running tests with mocha